src/Aquifer.Jobs/Subscribers/ScoreResourceContentVersionSimilarityMessageSubscriber.cs

```diff
@@ -0,0 +1,246 @@
+using Aquifer.Common.Messages;
+using Aquifer.Common.Messages.Models;
+using Aquifer.Common.Tiptap;
+using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class ScoreResourceContentVersionSimilarityMessageSubscriber(
+    AquiferDbContext _dbContext,
+    ILogger<ScoreResourceContentVersionSimilarityMessageSubscriber> _logger)
+{
+    [Function(nameof(ScoreResourceContentVersionSimilarity))]
+    public async Task ScoreResourceContentVersionSimilarity(
```

Comment:
I renamed all of the subscriber functions to end with `MessageSubscriber` to bring clarity when we see the names in the Azure Portal.
```suggestion
    public async Task ScoreResourceContentVersionSimilarityMessageSubscriber(
```
When you make this change you'll need to also rename the class (or choose a different root name for the Function).